plugins/api-docs/src/options.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { usePluginOptions } from '@backstage/core-plugin-api/alpha';
+
+export type ApiDocsPluginOptions = {
+  requestInterceptor: (req: any) => any;
+};
+
+/** @ignore */
+export type ApiDocsInputPluginOptions = {
+  requestInterceptor: (req: any) => any;
```

Comment:
We'll need proper types here 😉

Reply:
Any suggestion on what type should I use here? I tried:
`import { Request } from 'express';`

but SwaggerUI doesn't like it. They have this on the source code but seems to be the same as `any`
```
interface Request {
    [k: string]: any;
}
```
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/swagger-ui-react/index.d.ts#L9C1-L11C2